cli/omnibus/config/software/kontena.rb

```diff
@@ -7,6 +7,7 @@
 dependency "libxslt"
 whitelist_file "./wrappers/sh/kontena"
 build do
+  gem "install rb-readline -v 0.5.4 --no-ri --no-doc"
```

Comment:
Ideally this should be a dependency of the `kontena-plugin-shell` gem, but apparently conditional dependencies for different platforms (like omnibus not having a working stdlib readline?) are difficult.